docs/api-reference/aggregation-layers/hexagon-layer.md

```diff
@@ -8,34 +8,125 @@ The `HexagonLayer` aggregates data into a hexagon-based heatmap. The color and h
 
 HexagonLayer is a [CompositeLayer](../core/composite-layer.md) and at the moment only works with `COORDINATE_SYSTEM.LNGLAT`.
 
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="language">
+  <TabItem value="js" label="JavaScript">
+
 ```js
+import {Deck} from '@deck.gl/core';
+import {HexagonLayer} from '@deck.gl/geo-layers';
+
+const layer = new HexagonLayer({
+  id: 'HexagonLayer',
+  data: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/sf-bike-parking.json',
+
+  extruded: true,
+  getPosition: d => d.COORDINATES,
+  getColorWeight: d => d.SPACES,
+  getElevationWeight: d => d.SPACES,
+  elevationScale: 4,
+  radius: 200,
+  pickable: true
+});
+
+new Deck({
+  initialViewState: {
+    longitude: -122.4,
+    latitude: 37.74,
+    zoom: 11
+  },
+  controller: true,
+  getTooltip: ({object}) => object && `Count: ${object.elevationValue}`,
+  layers: [layer]
+});
+```
+
+  </TabItem>
+  <TabItem value="ts" label="TypeScript">
+
+```ts
+import {Deck, PickingInfo} from '@deck.gl/core';
+import {HexagonLayer} from '@deck.gl/geo-layers';
+
+type BikeRack = {
+  ADDRESS: string;
+  SPACES: number;
+  COORDINATES: [longitude: number, latitude: number];
+};
+
+const layer = new HexagonLayer<BikeRack>({
+  id: 'HexagonLayer',
+  data: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/sf-bike-parking.json',
+
+  extruded: true,
+  getPosition: (d: BikeRack) => d.COORDINATES,
+  getColorWeight: (d: BikeRack) => d.SPACES,
+  getElevationWeight: (d: BikeRack) => d.SPACES,
+  elevationScale: 4,
+  radius: 200,
+  pickable: true
+});
+
+new Deck({
+  initialViewState: {
+    longitude: -122.4,
+    latitude: 37.74,
+    zoom: 11
+  },
+  controller: true,
+  getTooltip: ({object}: PickingInfo<BikeRack>) => object && `Count: ${object.elevationValue}`,
```

Comment:
The typing here is incorrect - aggregation layers currently do not export any types for their picking infos. I want to keep this PR strictly about docs but this needs to be resolved soon.